### ISSUE TITLE
 Get rid of default chest at spawn

### ISSUE THREAD

**Issue by halvalla01** [MelanX/SkyblockBuilder#177]
### Describe your idea

I'm making a pack, and when I spawn on the skyblock world type it gives me the classic skyblock starter chest (cobble gen, some plants, nether portal, that kinda thing). I'd like a way to remove that chest entirely in the configs or somewhere else in my instance. 
If this is not a feature, will it ever be implemented? I'm on 1.19.2

**Comment by MelanX** [comment]
You can use whatever island you want. You can remove the chest, adjust the chest, replace the tree, and fill the world with lava. Most of that since 1.16.3, everything else since 1.17.
For reference, look at the wiki: https://wiki.chaotictrials.de/skyblock-builder/1.19.x/packdev/packdev/#creating-a-custom-skyblock-island
You can also find some examples to download.

tl;dr just replace `config/skyblockbuilder/templates/default.nbt` with a template you'd like to use.
